.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,3 +26,6 @@ List all PRs related to resolving a ticket. For instance, if you submitted a PR
 - Has server changes (please link here)
 - Has redux changes (please link here)
 - Has mobile changes (please link here)
+
+#### Screenshots
+[If the PR includes UI changes, include screenshots/GIFs.]
```

Comment:
What do you think about using the comment style from above?
```suggestion
<!--
[If the PR includes UI changes, include screenshots/GIFs.]
-->
```

Reply:
Good idea. Fixed!